### PR TITLE
Added logic for non minified files when deploying to state

### PR DESCRIPTION
**Pull request by rodrigopavezi**
Fixed jshint issues

@stulees please review. cheers

### CODE REVIEW

**Comment by stulees** [comment]
@rodrigopavezi An easier way to get around our functions being referenced before being declared is to set the value of `latedef` in the .jshintrc file to be "nofunc":

```
"latedef"      : "nofunc"
```

I'm doing this in my Image widget changes and it prevents from having to rearrange all our code which I'm finding problematic in Image.

**Comment by rodrigopavezi** [comment]
@stulees good call. Changed to that. please review. cheers

**Comment by rodrigopavezi** [comment]
@stulees ah, I think I changed that last time but forgot to run gulp build. so the change got deployed from the src and it is right. When I saw that it scared out me. cheers

File: src/config/prod.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 /* global config: true */
 /* exported config */
+var config;
 if (typeof config === "undefined") {
-  var config = {
```
Comment (stulees) [inline src/config/prod.js]: @rodrigopavezi Not sure which way is better for us, but in my Image changes, I've changed the use of `config` to be done as so:

```
/* exported config */
if (typeof angular !== "undefined") {
  angular.module("risevision.common.i18n.config", [])
    .constant("LOCALES_PREFIX", "locales/translation_")
    .constant("LOCALES_SUFIX", ".json");
}

var config = {
  STORAGE_ENV: "prod"
};
```
Just declaring it and assigning it once and omitting the check of `undefined` and the jshint comment for global. What you think?

Comment (rodrigopavezi) [inline src/config/prod.js]: @stulees that actually makes sense as it does not call a method from config so there is no need for checking if it is undefined. I will make the change. cheers

**Comment by rodrigopavezi** [comment]
@stulees changes made. please review. cheers

**Comment by stulees** [comment]
@rodrigopavezi Looks good :+1:

**Pull request merged**